assets/js/plugins/KimaiDatatable.js

```diff
@@ -79,11 +82,26 @@ export default class KimaiDatatable extends KimaiPlugin {
                     jQuery(html).find(contentArea)
                 );
                 durations.updateRecords();
+                self.fixDropdowns();
             },
             error: function(xhr, err) {
                 form.submit();
             }
         });
 
     }
+
+    fixDropdowns() {
+        // HACK: show dropdown menu upwards, if it "leaves" the page
+        jQuery(this.selector + ' [data-toggle=dropdown]').each(function() {
+            let parent = jQuery(this).parent();
+            let menu = parent.find('.dropdown-menu');
+
+            if (parent && menu) {
+                if ((parent.offset().top + parent.outerHeight() + menu.outerHeight()) > jQuery(document).height()) {
+                    parent.addClass('dropup');
```

Comment:
I'd say you should remove the "dropdown" class in that case. At least it works the same in my test and the example at 
https://getbootstrap.com/docs/3.4/components/#dropdowns-example
looks as if the dropdown class is not required.

Reply:
works for me as well